From: Rob Evers <revers@redhat.com>
Date: Tue, 5 Oct 2010 14:24:53 -0400
Subject: [scsi] lpfc: remove unused variables
Message-id: <1286288695-20754-22-git-send-email-revers@redhat.com>
Patchwork-id: 28611
O-Subject: [RHEL5.6 PATCH 21/23] Remove unused variables that were removed from
	upstream submission
Bugzilla: 619917
RH-Acked-by: Tomas Henzl <thenzl@redhat.com>

From: Rob Evers on behalf of Emulex <revers@redhat.com>

https://bugzilla.redhat.com/show_bug.cgi?id=619917

diff --git a/drivers/scsi/lpfc/lpfc.h b/drivers/scsi/lpfc/lpfc.h
index be6403d..6b088a8 100644
--- a/drivers/scsi/lpfc/lpfc.h
+++ b/drivers/scsi/lpfc/lpfc.h
@@ -924,8 +924,6 @@ struct lpfc_hba {
 	struct list_head fcf_conn_rec_list;
 	uint32_t iocb_cnt;
 	uint32_t iocb_max;
-	uint32_t ns_cnt;
-	uint32_t ns_max;
 	atomic_t sdev_cnt;
 };
 
diff --git a/drivers/scsi/lpfc/lpfc_ct.c b/drivers/scsi/lpfc/lpfc_ct.c
index 8f60982..128c26c 100644
--- a/drivers/scsi/lpfc/lpfc_ct.c
+++ b/drivers/scsi/lpfc/lpfc_ct.c
@@ -584,8 +584,6 @@ lpfc_cmpl_ct_cmd_gid_ft(struct lpfc_hba *phba, struct lpfc_iocbq *cmdiocb,
 	struct lpfc_nodelist *ndlp;
 	int rc;
 
-	phba->ns_cnt--;
-
 	/* First save ndlp, before we overwrite it */
 	ndlp = cmdiocb->context_un.ndlp;
 
@@ -745,8 +743,6 @@ lpfc_cmpl_ct_cmd_gff_id(struct lpfc_hba *phba, struct lpfc_iocbq *cmdiocb,
 	uint8_t fbits;
 	struct lpfc_nodelist *ndlp;
 
-	phba->ns_cnt--;
-
 	did = ((struct lpfc_sli_ct_request *) inp->virt)->un.gff.PortId;
 	did = be32_to_cpu(did);
 
@@ -932,8 +928,6 @@ lpfc_cmpl_ct_cmd_rft_id(struct lpfc_hba *phba, struct lpfc_iocbq *cmdiocb,
 	IOCB_t *irsp = &rspiocb->iocb;
 	struct lpfc_vport *vport = cmdiocb->vport;
 
-	phba->ns_cnt--;
-
 	if (irsp->ulpStatus == IOSTAT_SUCCESS) {
 		struct lpfc_dmabuf *outp;
 		struct lpfc_sli_ct_request *CTrsp;
@@ -955,8 +949,6 @@ lpfc_cmpl_ct_cmd_rnn_id(struct lpfc_hba *phba, struct lpfc_iocbq *cmdiocb,
 	IOCB_t *irsp = &rspiocb->iocb;
 	struct lpfc_vport *vport = cmdiocb->vport;
 
-	phba->ns_cnt--;
-
 	if (irsp->ulpStatus == IOSTAT_SUCCESS) {
 		struct lpfc_dmabuf *outp;
 		struct lpfc_sli_ct_request *CTrsp;
@@ -978,8 +970,6 @@ lpfc_cmpl_ct_cmd_rspn_id(struct lpfc_hba *phba, struct lpfc_iocbq *cmdiocb,
 	IOCB_t *irsp = &rspiocb->iocb;
 	struct lpfc_vport *vport = cmdiocb->vport;
 
-	phba->ns_cnt--;
-
 	if (irsp->ulpStatus == IOSTAT_SUCCESS) {
 		struct lpfc_dmabuf *outp;
 		struct lpfc_sli_ct_request *CTrsp;
@@ -1001,8 +991,6 @@ lpfc_cmpl_ct_cmd_rsnn_nn(struct lpfc_hba *phba, struct lpfc_iocbq *cmdiocb,
 	IOCB_t *irsp = &rspiocb->iocb;
 	struct lpfc_vport *vport = cmdiocb->vport;
 
-	phba->ns_cnt--;
-
 	if (irsp->ulpStatus == IOSTAT_SUCCESS) {
 		struct lpfc_dmabuf *outp;
 		struct lpfc_sli_ct_request *CTrsp;
@@ -1023,8 +1011,6 @@ lpfc_cmpl_ct_cmd_da_id(struct lpfc_hba *phba, struct lpfc_iocbq *cmdiocb,
 {
 	struct lpfc_vport *vport = cmdiocb->vport;
 
-	phba->ns_cnt--;
-
 	/* even if it fails we will act as though it succeeded. */
 	vport->ct_flags = 0;
 	lpfc_cmpl_ct(phba, cmdiocb, rspiocb);
@@ -1038,8 +1024,6 @@ lpfc_cmpl_ct_cmd_rff_id(struct lpfc_hba *phba, struct lpfc_iocbq *cmdiocb,
 	IOCB_t *irsp = &rspiocb->iocb;
 	struct lpfc_vport *vport = cmdiocb->vport;
 
-	phba->ns_cnt--;
-
 	if (irsp->ulpStatus == IOSTAT_SUCCESS) {
 		struct lpfc_dmabuf *outp;
 		struct lpfc_sli_ct_request *CTrsp;
@@ -1272,8 +1256,6 @@ lpfc_ns_cmd(struct lpfc_vport *vport, int cmdcode,
 		lpfc_debugfs_disc_trc(vport, LPFC_DISC_TRC_CT,
 			"Issue CT cmd:    cmd:x%x did:x%x",
 			cmdcode, ndlp->nlp_DID, 0);
-		if (++phba->ns_cnt > phba->ns_max)
-			phba->ns_max = phba->ns_cnt;
 		return 0;
 	}
 	rc=6;
